robolectric/build.gradle

```diff
@@ -41,10 +41,7 @@ dependencies {
     testImplementation libs.truth.java8.extension
     testImplementation libs.mockito
     testImplementation libs.hamcrest.junit
-    testImplementation "androidx.test:core:$axtCoreVersion@aar"
-    testImplementation "androidx.test.ext:junit:$axtJunitVersion@aar"
     testImplementation "androidx.test.ext:truth:$axtTruthVersion@aar"
-    testImplementation "androidx.test:runner:$axtRunnerVersion@aar"
```

Comment:
The `axtRunnerVersion` variable can be removed, no?

Reply:
What I meant is that this specific variable (`axtRunnerVersion`) is no longer used in the Robolectric source code. That's why I was suggesting to remove it

Reply:
In the current master (ab2d2cf5ba98d61541433c3bb3ec533b254ca98a), this is still the latest usage of `axtRunnerVersion`. So I'd say that it can be removed 🙂